OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -766,16 +766,25 @@ static bool Team(S server, Connection conn, Session.Client client, string s)
 		static bool ToggleSpawn(S server, Connection conn, Session.Client client, string s)
 		{
 			if (!client.IsAdmin)
-				return false;
+			{
+				server.SendOrderTo(conn, "Message", "Only admins can disable spawn points.");
+				return true;
+			}
 
-			var spawn = Exts.ParseIntegerInvariant(s);
-			if (spawn == 0)
+			var spawnPoint = Exts.ParseIntegerInvariant(s);
+			if (spawnPoint == 0)
 				return true;
 
-			if (server.LobbyInfo.DisabledSpawns.Contains(spawn))
-				server.LobbyInfo.DisabledSpawns.Remove(spawn);
+			if (server.LobbyInfo.Clients.Any(cc => cc.SpawnPoint == spawnPoint))
```

Comment:
I copied the relevant bit of this check from the `Spawn` method; the variable used there was `spawnPoint` so I adjusted to the same here.